client/components/parts/SignUp.jsx

```diff
@@ -0,0 +1,43 @@
+import React, {PureComponent} from 'react'
+
+import SignUpForm from './SignUpForm'
+
+// use when state and lifecycle functions are needed
+export default class SignUp extends PureComponent {
+  constructor(props) {
+    super(props)
+    this.state = {}
+  }
+
+  static propTypes = {
+    active: React.PropTypes.bool,
+    close: React.PropTypes.func
```

Comment:
Should either of these be required when inserting the component? 
If so, you can add .isRequired after the .bool or .func